File: algorithmsOnArray/bubbleSort.js
let list =[8,434,232,334,22]
let lengthOfList=list.length
function bubbleSort(list){
    for(let i=0;i<lengthOfList-1;i++){
        for(let j=0;j<lengthOfList-i-1;j++){
            if(list[j]>list[j+1]){
                [list[j],list[j+1]]=[list[j+1],list[j]]

            }
        }
    }
    return list
}

console.log(bubbleSort(list))

File: algorithmsOnArray/bubbleSort.py
list=[3,5,2,6,2]
lengthOfList=len(list)
for i in range(lengthOfList-1):
    swapped=False
    for j in range(lengthOfList-i-1):
      
        if list[j]>list[j+1]:
            list[j],list[j+1]=list[j+1],list[j]
            swapped=True
    if not swapped:
        break
print(" the list is swapped through bubble sort",list)
    